seqr/utils/file_utils.py

```diff
@@ -35,7 +35,8 @@ def file_iter(file_path, byte_range=None, raw_content=False):
         for line in _google_bucket_file_iter(file_path, byte_range=byte_range, raw_content=raw_content):
             yield line
     else:
-        with open(file_path) as f:
+        mode = 'rb' if raw_content else 'r'
+        with open(file_path, mode) as f:
```

Comment:
Just a reminder that not all the raw files can be read line by line. For example, the  `.xls` files.